packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -90,3 +90,13 @@ export const menuIconStyleSet = {
     width: '1.25rem'
   }
 };
+
+/**
+ * @private
+ */
+export const menuSubIconStyleSet = {
+  root: {
+    height: 'calc(100% - 10px)',
```

Comment:
Why `- 10px`? Is 10px this the size of the current seen icon? Contoso is able to override icons so this fix may not scale well

Reply:
So the idea here is to use a similar offset to the main Icon to have it match in alignment. I agree that this will possibly cause issues with custom icons but if set to use the same styles as the main Icon for the context menu it appears slightly higher.